sound/components/virt.c

```diff
@@ -0,0 +1,248 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <microkit.h>
+#include <sddf/sound/queue.h>
+
+#define DRIVER_CH 3
```

Comment:
The `DRIVER_CH` should always be 0 and that way the `DRIVER_CH` does not depend on the number of clients.

Reply:
Done

Reply:
Just letting you know this is not the case for serial at the moment